pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -159,3 +173,20 @@ func (r *ReconcileNode) cephPodList() ([]corev1.Pod, error) {
 
 	return cephPods, nil
 }
+
+func getImageVersion(image string) (*version.CephVersion, error) {
+	retries := 0
+	for {
```

Comment:
Even though it will be functionally equivalent, i'd avoid the appearance of an infinite loop. For example, in simplified terms:
```
for i:=0; i < maxRetries; i++ {
  if successful {
    return version, nil
  } 
}
getVersionFailed = true
return nil, fmt.Errorf("ceph version not found for image %q", image)
```

Reply:
Fair enough. I was following a pattern elsewhere in the codebase, but I'm happy to do this instead.